src/app/modules/wallet/pages/account-list/account-list.component.ts

```diff
@@ -53,14 +63,12 @@ export class AccountListComponent implements OnInit {
   }
 }
 
-function createNewUser(id: number): UserData {
-  const name = NAMES[Math.round(Math.random() * (NAMES.length - 1))] + ' ' +
-      NAMES[Math.round(Math.random() * (NAMES.length - 1))].charAt(0) + '.';
+function createNewUser(id: number, name: string): UserData {
```

Comment:
Can we remove all this mock logic and instead just use the accounts and display real data?